JavaAssignments/src/ankush/CountVisibleObject.java

```diff
@@ -0,0 +1,27 @@
+package ankush;
+
+public class CountVisibleObject {
+	int printTree()
+	{
+		int treecount=1;
+		DynamicInputArray ia=new DynamicInputArray();
+		int temp[]=ia.intArray();
+		int maxValue=temp[0];
+		for (int index=0;index<temp.length;index++)
```

Comment:
More specifically, you have already taken 0th index tree size in maxValue, so dont need to start index from 0, for loop will iterate one time extra in this case. consider 0th index iteration. you should start from 1 as maxValue is already indicating 0th index value. 

NOTE : it will not make visible difference still we should optimize code in a way to iterate loop only required time.